ui/packages/shared/profile/src/utils.ts

```diff
@@ -50,3 +50,11 @@ export const truncateString = (str: string, num: number): string => {
 
   return str.slice(0, num) + '...';
 };
+
+export const truncateStringReverse = (str: string, num: number): string => {
```

Comment:
I think this is a good example of something that could live in the @parca/utils package once that is in place :) cc @yomete